pkg/api/rules_engine/rules_engine.go

```diff
@@ -0,0 +1,30 @@
+package rules_engine
+
+import (
+	"context"
+	"io"
+
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/utils"
+	"go.uber.org/zap"
+)
+
+func (c *Client) Delete(ctx context.Context, edgeApplicationID int64, phase string, ruleID int64) error {
```

Comment:
Wouldn't a `delete.go` file be more interesting?

Reply:
Hmmm 
Creating a file for a single function doesn't seem too useful/reusable.

Reply:
I think it's better for reading and searching code, but that's not a problem. LGTM 👍🏽